sdk/storage/azure-storage-file-share/azure/storage/fileshare/_directory_client.py

```diff
@@ -66,6 +66,10 @@ class ShareDirectoryClient(StorageAccountHostsMixin):
         - except in the case of AzureSasCredential, where the conflicting SAS tokens will raise a ValueError.
         If using an instance of AzureNamedKeyCredential, "name" should be the storage account name, and "key"
         should be the storage account key.
+    :keyword bool allow_trailing_dot: If true, the trailing dot will not be trimmed from the target URI.
+        Default value is None.
```

Comment:
Can remove the "Default value is None" from all these. It's an optional keyword so its assumed :).